.pre-commit-config.yaml

```diff
@@ -7,21 +7,23 @@ repos:
           - "--unsafe"
       - id: end-of-file-fixer
       - id: trailing-whitespace
-  - repo: https://github.com/psf/black
-    rev: 23.1.0
+        exclude: "README.md"
```

Comment:
I'm very confused.  Why can't we remove the trailing whitespace from README.md?

Reply:
Markdown treats two consecutive spaces as `<b/>` and this is used in README to generate nicely formatted output. I think we'll want to drop `README` in the future because the same contents is in `doc/index.rst`. I'll make a ticket for myself to improve documentation.

Reply:
I'm pretty sure GitHub markdown allows a `\` at the end of the line to be equivalent to the trailing spaces.

Reply:
I switched to `\`.